src/main/java/seedu/ultron/logic/parser/StatusCommandParser.java

```diff
@@ -19,7 +20,7 @@ public class StatusCommandParser implements Parser<StatusCommand> {
     public StatusCommand parse(String arg) throws ParseException {
         String trimmedArg = arg.trim();
         String[] nameKeywords = trimmedArg.split("\\s+");
```

Comment:
No reason to split and check input here. All argument validity checks should be done in predicate/model constructors

Reply:
Use the parseStatus method in ParserUtil